tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -277,7 +282,9 @@ def run(self):
                 # it can if self._run fails to set self.result due to raising
                 self.result = False, ("INTERNAL-ERROR", "self._run didn't set a result")
             else:
-                self.result = False, ("EXTERNAL-TIMEOUT", None)
+                message = "Waiting on browser:\n"
+                message += "".join(traceback.format_stack(sys._current_frames()[executor.ident]))
```

Comment:
Comment?

Reply:
Added some comment. Let me knows if this is good enough (given the context with `executor` defined a few lines above, anything more seems like you're literally handholding through the line).